src/strings/csv/es.csv

```diff
@@ -1131,16 +1199,27 @@ RECEIVED_DATE,Fecha de recepción,
 RECEIVED_ON,Recibido/a el,
 RECEIVING_DATE_REQUIRED,Fecha de recepción *,
 RECEIVING_NURSERY_REQUIRED,Vivero de recepción *,
+RECOMMENDED,Recomendado,
 RECORDED_PLANTS,Plantas registradas,
 RECORDED_SPECIES,Especies registradas,
 RECORDING_DATE,Fecha de registro,
 RECORDING_DATE_ERROR,Debe ser posterior a la fecha de inicio,
+REFERENCE,Referencia,
 REFRESH,Actualizar,
 REFRESH_DATA,Actualizar datos,
 REFRIGERATOR_1,Refrigerador 1,
 REFRIGERATOR_2,Refrigerador 2,
 REFRIGERATOR_3,Refrigerador 3,
 REGION,Región,
+REGION_ANTARCTICA,Antártida,
+REGION_EAST_ASIA_PACIFIC,Asia Oriental y Pacífico,
+REGION_EUROPE_CENTRAL_ASIA,Europa y Asia Central,
+REGION_LATIN_AMERICA_CARIBBEAN,América Latina y el Caribe,
+REGION_MIDDLE_EAST_NORTH_AFRICA,Oriente Medio y África del Norte,
+REGION_NORTH_AMERICA,América del Norte,
+REGION_OCEANIA,Oceanía,
+REGION_SOUTH_ASIA,Asia del Sur,
+REGION_SUB_SAHARAN_AFRICA,África subsahariana,
```

Comment:
Some of these don't match BE translations.
```
src/main/resources/i18n/Enums_es.properties:public.Region.Antarctica=La Antártida
src/main/resources/i18n/Enums_es.properties:public.Region.EastAsiaPacific=Asia Oriental y el Pacífico
src/main/resources/i18n/Enums_es.properties:public.Region.LatinAmericaCaribbean=Latinoamérica y el Caribe
src/main/resources/i18n/Enums_es.properties:public.Region.NorthAmerica=Norteamérica
src/main/resources/i18n/Enums_es.properties:public.Region.SouthAsia=Sudeste Asiático

REGION_ANTARCTICA,Antártida,
REGION_EAST_ASIA_PACIFIC,Asia Oriental y Pacífico,
REGION_LATIN_AMERICA_CARIBBEAN,América Latina y el Caribe,
REGION_NORTH_AMERICA,América del Norte,
REGION_SOUTH_ASIA,Asia del Sur,
```

We may need to update these in `phrase`.

Reply:
I don't have access to Phrase, would you be able to edit the values @nickgraz ?

Reply:
I have access, but just noticed I can't see the projects. I will fix when I get access.

Reply:
Thanks @nickgraz sounds good.